test/matchers/dimension_validator_matcher_test.rb

```diff
@@ -143,4 +143,62 @@ class ActiveStorageValidations::Matchers::DimensionValidatorMatcher::Test < Acti
     matcher.width_min 800
     refute matcher.matches?(Project.new)
```

Comment:
Regarding my comment below, I was referring to this line. But you're right and we should probably pass the class instead of an object everywhere.